packages/ember/tests/dummy/app/pods/index/template.hbs

```diff
@@ -0,0 +1,75 @@
+<DocsHeader @prefix='' @name={{milestones-icon}} />
+
+<div local-class="hero">
+  <DocsHero
+    @prefix=''
+    @heading='Milestones'
+    @byline='Navigate concurrent code with ease.'
+  />
+</div>
+
+<div class='docs-container docs-md' local-class="content">
+  <section class='docs-max-w-lg docs-mx-auto docs-pb-8 docs-mt-16'>
+    <h2>What's This?</h2>
+
+    The <code>@milestones</code> packages provide a set of tools for navigating concurrent code in testing and
+    development. Milestones act as named synchronization points, and they give you the ability to change the behavior
+    of annotated code during testing, skipping pauses or mocking out results.
+  </section>
+
+  <h2>Example</h2>
```

Comment:
The short introduction you have at https://github.com/salsify/milestones/pull/19/files#diff-7eceec1e73cd77044ad8827b2cc22c9fR9 would be helpful here, even with the "Not sure what you're looking at?" below.

Even though I've seen the playground before, that intro helped me intuitively figure out what I was looking at, while The Playground section was more in-depth than I needed right away.